### DISCUSSION TITLE
Worker Creates New Folder each time has new job

### DISCUSSION THREAD

**Discussion by imdevopx** [windmill-labs/windmill#4896] [Questions]
worker creates new folder in server /tmp/windmill/<worker-hostname>/ every time it has new job even same workflow. so it accumulates and so on, server gets full disk.. is this a normal behavior? any suggestions how to fix this kind of behavior?

**Comment by uael** [comment]
Which version are you using ? I'm not sure if its related but there was an issue regarding flow cached values under `tmp/windmill/flow/`, it has been fixed in `1.436`

**Reply by imdevopx** [reply]
1.434.2 i will try to update it.. and will get back to you. thanks.

**Reply by imdevopx** [reply]
hi @uael seems we turned-on the Keep Job Directories for debug.. that causes our issue. i have some another question though, on that same workflow if we run it. The Workers started to status DEAD but eventually goes back to Alive, i see that Last ping is around 120s. any idea what causes it?
